pkg/manager/service.go

```diff
@@ -163,10 +163,28 @@ func (m *Manager) Delete(ctx context.Context, req *rpc.DeleteRequest) (resp *emp
 		return nil, err
 	}
 
+	if err := m.waitForFileDeleted(req.Name, req.Uuid, 3); err != nil {
```

Comment:
Is 3 seconds too short?

Reply:
1. Currently, the BackingImageManager server and the file-syncing server are on the same node. And the deletion in the file-syncing server is not an async operation. Hence I think 3 seconds is enough, which is the same the monitor interval.
2. I don't want to block the manager deletion call for a long time. This is a sync call.